frontend/app/(pages)/projektbeschreibung/page.tsx

```diff
@@ -52,28 +56,21 @@ export default async function ProjectDescription() {
             Dialog mit diesen.
           </li>
         </ul>
-      </div>
-      <h1 className="headingWithBar w-sm-75 m-auto">Über uns</h1>
-      <div className="w-sm-50 m-auto py-3">
-        <h4>LocalMonitoring wird ehrenamtlich von engagierten Bürger:innen der jeweiligen Stadt betrieben.</h4>
-        LocalMonitoring ist ein Projekt der Initiative LocalZero, dem Netzwerk für kommunale Klimaneutralität unter dem
-        Dach von{" "}
-        <a
-          href="https://germanzero.de/"
-          target="_blank"
-        >
-          GermanZero
-        </a>
-        . Teams von LocalZero sind in mehr als 90 Städten in Deutschland aktiv. In über 40 Städten haben sie bereits
-        Beschlüsse für Klimaneutralität bis spätestens 2035 bewirkt.
+
+        <h1 className="headingWithBar">Über uns</h1>
+
+        <h4>Ehrenamts-Power</h4>
+        <p>LocalMonitoring ist ein Projekt der Initiative LocalZero, dem Netzwerk für kommunale Klimaneutralität unter dem Dach von <a href="https://germanzero.de/" target="_blank">GermanZero</a>. Dennoch wird das Tool ehrenamtlich von engagierten Bürger:innen entwickelt und inhaltlich gepflegt. Teams von LocalZero sind in mehr als 90 Städten in Deutschland aktiv. In über 40 Städten haben sie bereits
```

Comment:
I would additionally link to LocalZero:
```suggestion
        <p>LocalMonitoring ist ein Projekt der Initiative <a href="https://localzero.net/" target="_blank">LocalZero</a>, dem Netzwerk für kommunale Klimaneutralität unter dem Dach von <a href="https://germanzero.de/" target="_blank">GermanZero</a>. Dennoch wird das Tool ehrenamtlich von engagierten Bürger:innen entwickelt und inhaltlich gepflegt. Teams von LocalZero sind in mehr als 90 Städten in Deutschland aktiv. In über 40 Städten haben sie bereits
```